packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -77,6 +79,7 @@ export type GridConfig<T extends { __typename?: string }> = {
     delete?: boolean;
     copyPaste?: boolean;
     readOnly?: boolean;
+    initialSort?: Array<{ field: string; sort: GridSortDirection }>;
```

Comment:
Could we infer the available fields for sorting here? Maybe by an additional generic? @nsams @Ben-Ho is this something we should add?

Reply:
the correct way would be to infer form `GQLProductSortField` - the enum with the possible sort fields. That doesn't necessarily match the `GQLProduct` keys (although it will in most cases)

Currently we only have `GQLProduct`, which is easy to use (vs. a additional `GQLQueryproductsArgs` generic argument)

So I'm leaning towards leaving this as string.